sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -572,6 +572,13 @@ object SQLConf {
       "disable logging or -1 to apply no limit.")
     .createWithDefault(1000)
 
+  val WHOLESTAGE_MAX_LINES_PER_FUNCTION = buildConf("spark.sql.codegen.maxLinesPerFunction")
+    .internal()
+    .doc("The maximum lines of a function that will be supported before" +
+      " deactivating whole-stage codegen.")
```

Comment:
> The maximum lines of a single Java function generated by whole-stage codegen. When the generated function exceeds this threshold, the whole-stage codegen is deactivated for this subtree of the current query plan. 

Could you also update the code comments in the other places based on my above update?